jetty-servlets/src/main/java/org/eclipse/jetty/servlets/CrossOriginFilter.java

```diff
@@ -63,16 +62,33 @@
  * <p>
  * Allowed origins can therefore be more complex expressions such as
  * https?://*.domain.[a-z]{3} that matches http or https, multiple subdomains
- * and any 3 letter top-level domain (.com, .net, .org, etc.).</dd>
- * 
+ * and any 3 letter top-level domain (.com, .net, .org, etc.).
+ * <p>
+ * Won't be used if <b>allowedOriginsRegex</b> is declared.
+ * </dd>
+ * <p>
+ * <dt>allowedOriginsRegex</dt>
+ * <dd>a new line separated list of origins regular expressions that are
```

Comment:
Do we really need new line separation rather than comma separation?   comma is not an allowed character in a domain name.

Reply:
As the field transport a list of regex, and comma can easily fit inside a regex I thought that it would be more natural to use other separator (like newline). 

Does any sense for the user to write a comma inside a regex that matches the origin? I does not makes sense.

But I took that approach because it the one that generate least surprise in my opinion, but can be changed.